Octokit.Reactive/Clients/ObservableDeploymentStatusClient.cs

```diff
@@ -69,10 +106,28 @@ public IObservable<DeploymentStatus> GetAll(string owner, string name, int deplo
         /// <param name="name">The name of the repository.</param>
         /// <param name="deploymentId">The id of the deployment.</param>
         /// <param name="newDeploymentStatus">The new deployment status to create.</param>
-        /// <returns></returns>
         public IObservable<DeploymentStatus> Create(string owner, string name, int deploymentId, NewDeploymentStatus newDeploymentStatus)
         {
+            Ensure.ArgumentNotNull(newDeploymentStatus, "newDeploymentStatus");
```

Comment:
`Ensure.ArgumentNotNullOrEmptyString` for `owner` and `name` while we're in here